centos-7-minimal.ks

```diff
@@ -2,29 +2,39 @@ sshpw --username=root --plaintext centos
 # Firewall configuration
 firewall --disabled
 selinux --disabled
-#Use network installation
+
+# Use network installation
 url --url="http://mirror.centos.org/centos/7/os/x86_64/"
 network --bootproto=dhcp --device=link --activate --onboot=on
 skipx
 rootpw --plaintext centos
 auth --useshadow --passalgo=sha512
+
 timezone --utc America/New_York
 bootloader --location=mbr --append="no_timer_check console=tty0 console=ttyS0,115200 net.ifnames=0 biosdevname=0"
 clearpart --all
-part / --size=8192 --fstype ext4
+part / --fstype="ext4" --size=10240
 
 #Repos
 repo --name=base --baseurl=http://mirror.centos.org/centos/7/os/x86_64/
 repo --name=updates --baseurl=http://mirror.centos.org/centos/7/updates/x86_64/
 repo --name=extras --baseurl=http://mirror.centos.org/centos/7/extras/x86_64/
+repo --name=adb --baseurl=http://mirror.centos.org/centos/7/atomic/x86_64/adb/
+
+user --name=docker --password=tcuser
 
 shutdown
 
 %packages  --excludedocs --instLangs=en
 @core
 kernel
+bash
 dracut
+shadow-utils
 docker
+syslinux
+centos-logos
+net-tools
```

Comment:
@hferentschik I am not sure why we need net-tools by default in the ISO. Can you please explain it little more.